image_explorer/image_explorer.py

```diff
@@ -287,16 +286,30 @@ def _inner_content(self, tag):
             return u''.join(html.tostring(e) for e in tag)
         return None
 
-    def _get_description(self, xmltree):
+    def _get_description(self, xmltree, absolute_urls=False):
         """
         Parse the XML to get the description information
         """
         description = xmltree.find('description')
         if description is not None:
-            return self._inner_content(description)
+            description = self._inner_content(description)
+            if absolute_urls:
+                description = self._change_relative_url_to_absolute(description)
+            return description
         return None
 
-    def _get_hotspots(self, xmltree):
+    def _change_relative_url_to_absolute(self, text):
+        if text:
+            try:
+                from parsel import Selector
+            except ImportError:
```

Comment:
On what versions of edx-platform this is going to happen?

Reply:
@ahmedaljazzar  If you will run it on xblock SDK or if you run the unit tests for this xblock then you will have this import error.

Reply:
I see, so this is because the library is not listed in the [requirements](https://github.com/edx-solutions/xblock-image-explorer/blob/master/requirements.txt) file. Adding it to the requirements will save us the uncertainty in the behavior of this method, and will clean the test.

Reply:
Okay. Got it!